set.mm

```diff
@@ -38055,6 +38055,35 @@ among classes ( ~ eq0 , as opposed to the weaker uniqueness among sets,
       UNABUJTVBUNSSHSVBULSUMSABCTABDTMSUEUFUGUH $.
   $}
 
+  ${
+    $d A x y $.  $d B x y $.  $d X x y $.  $d ph y $.  $d ch x $.  $d ps x y $.
```

Comment:
Even if it is not necessary for the proof, I'd mark the dummy variable y as disjoint from ch.